### PR TITLE
Section 8

### PR DESCRIPTION
**Pull request by dsamojlenko**
Add Section 8 - Supporting Documents

Note: I didn't add any validation, I'm not sure checkboxes are the right control for this, and the file upload field is not functional. Figure we'll work out an interaction model for this page later, this is just a recreation of the pdf.

### CODE REVIEW

File: routes/section8/schema.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const Schema = {
+  firstname: {
+    isLength: {
+      errorMessage: 'errors.firstname.length',
+      options: { min: 3, max: 200 },
```
Comment (CalvinRodo) [inline routes/section8/schema.js]: Usually just safer not to validate names.
https://en.wikipedia.org/wiki/Cy_Young

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
Names should be minimum 1 character in length if validating.

Also getting a 500 error when submitting the page in the branch.

File: routes/section8/schema.js (hunk 1 above)
Comment (CalvinRodo) [reply routes/section8/schema.js]: Okay just realized that was a nickname so I'm not as clever as I thought but I'm sure someone has a 2 letter name out there

Comment (dsamojlenko) [reply routes/section8/schema.js]: lol, there's no name field. This was the sample validation from the route template, should have just been removed... now it is.

Comment (dsamojlenko) [reply routes/section8/schema.js]: The error was due to missing the csrf token field. Added it, and have also submitted a PR to the node starter repo adding that to the route template.

**Review by CalvinRodo — DISMISSED** [review]

**Review by jeffmaher — APPROVED** [review]

**Review by CalvinRodo — COMMENTED** [review]
🥇

**Pull request merged**